src/blog/2024/04/dashboard-1-8-0.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Dashboard 2.0: Milestones, PWA and New Components"
+subtitle: Checkout all the great content that's been added to Dashboard 2.0 in the past few weeks and the new (in-preview) Vuetify components we've made available in UI Template nodes.
+description: Checkout all the great content that's been added to Dashboard 2.0 in the past few weeks and the new (in-preview) Vuetify components we've made available in UI Template nodes.
+date: 2024-04-26
+authors: ["joe-pavitt"]
+image: /blog/2024/03/images/getting-started-with-dashboard-2.png
+tags:
+    - posts
+    - releases
+    - community
+    - changelog
+    - dashboard
+    - node-red Dashboard 2.0
+    - node-red visualization
+    - progressive web app
+    - Vuetify
+---
+
+With a new release of Node-RED Dashboard 2.0 we have plenty of new fixes and improvements being added to the project. In this post, we'll deep dive into community contributions, PWA support, new Vuetify components, and the rest of the great work published in this latest release.
+
+<!--more-->
+
+## Community Contributions
+
+I firstly wanted to take this opportunity to point out a big milestone that I'm very proud to see in Node-RED Dashboard 2.0.
+
+This release marks the first time we've had more contributions in a single release from the community, than from FlowFuse employees. I think this is a testament to the community, and a big milestone in validating the success, and popularity, of Dashboard 2.0 in the wider Node-RED community.
+
+So thank you very much [@BartButenaers](https://github.com/bartbutenaers), [@Ek1nox](https://github.com/Ek1nox), [@fullmetal-fred](https://github.com/fullmetal-fred) and [@cgjgh](https://github.com/cgjgh) for for great efforts and initiative in improving Node-RED Dashboard 2.0.
+
+For anyone else that's interested in contributing to the project, please do reach out, and we'll be happy to help you get started. We also have a [Contributing Guide](https://dashboard.flowfuse.com/contributing/) if you want to dive straight in.
+
+## Progressive Web App (PWA) Support
+
+The biggest community contribution we saw in this release was the addition of Progressive Web App (PWA) support. This feature was added by [@cgjgh](https://github.com/cgjgh), and allows you to install your Node-RED Dashboard 2.0 applications directly onto your platform, including Windows, iOS and Android.
+
+This work will give your Dashboard's a much more native/natural feel when running on your own machines, and mean you no longer need to go via your browser to access your applications.
```

Comment:
Link to docs on how to achieve this and use it?

Reply:
We actually don't have any currently, I'll see what I can put together.